content/departments/product-engineering/engineering/process/releases/index.md

```diff
@@ -120,6 +116,117 @@ The release captain has unlimited power to make changes to the release branch to
 
 Most issues are non-blocking. Fixes to non-blocking issues can be fixed in `main` by the code owner who can then `git cherry-pick` those commits into the release branch with the approval of the release captain. Alternatively, broken features can be reverted out of the release branch or disabled via feature flags if they aren't ready or are too buggy.
 
+## Patch release process
+
+> As of Jan 31, 2022, the Delivery team still owns the release process hence the role of Release Captain, `us` in the process below is always refering to the Delivery team. However, this may change in the future when we transit to a rotation base Release Captain.
+
+### 1) Kickstart a patch release
+
+CE or the products team will start requesting a patch release by submitting a [patch release request]. The patch release to [revert poor onboarding UX change] is a good example.
+
+Upon the Delivery team or the Release Captain receives the patch release request and we have decided to rollout a new patch release, the Release Captain should follow the instruction in the [patch release request] issue to kickstart the patch release process.
```

Comment:
Maybe we should add another section detailing how to decide whether to accept the patch - who gets to decide? Who needs to be part of the conversation, and who gets notified? What's the timeframe?

I don't think we have good answers - pretty much the release captain says 'yes' and immediately works on the patch. CS generally gets looped in by either the dev requesting the patch or the release captain, since a patch almost always causes additional work for them.